src/scripts/components/PlayerControls.js

```diff
@@ -409,29 +404,31 @@ class PlayerControls extends Component<Props, State> {
               </VolumeBarWrapper>
             </LeftControls>
             <RightControls>
-              <ControlButtonWrapper>
-                <IconButton
-                  color={
-                    activePlugin === PLAYER_PLUGIN.PLAYBACK_LEVELS
-                      ? Colors.purple
-                      : undefined
-                  }
-                  data-test-id="playback-levels-button"
-                  disabled={!playbackLevels.size}
-                  icon={qualityIcon}
-                  onClick={() => {
-                    toggleActivePlugin(PLAYER_PLUGIN.PLAYBACK_LEVELS)
-                  }}
-                />
+              <ControlButtonWrapper
+                accent={activePlugin === PLAYER_PLUGIN.PLAYBACK_LEVELS}
+                warn={activePlugin !== PLAYER_PLUGIN.PLAYBACK_LEVELS}
```

Comment:
So as not to duplicate logic, could the logic be abstracted into an instance method?

Reply:
Done https://github.com/Paratii-Video/paratii-portal/pull/769/commits/7901ddd97ee96adb3e23a225cc60947fc8592fe0